nav_msgs/Odometry waypoint
---
bool success